install.sh

```diff
@@ -2,10 +2,10 @@
 set -eu
 
 # Install the JavaScript dependencies
-yarn
+(cd node && yarn)
```

Comment:
1. > Actually, srcjs -> node/src. I wanted to put all files related to Node into a single directory to keep things tidy. I figured node suits this intention better and I didn't want to have srcjs/src.

How about `js` and `js/src` ? I don't like `node` because this suggests there's some node application and basically node is not the important point here. This is the JS part of our codebase, not Node part of our codebase -- we're just using node to build the JS part of the frontend


2. since we're making changes to this path and it is repeated, I suggest making this a variable

Reply:
Done.